app/controllers/nominations/nominate_induction_coordinator_controller.rb

```diff
@@ -1,22 +1,28 @@
 # frozen_string_literal: true
 
 class Nominations::NominateInductionCoordinatorController < ApplicationController
-  def new
-    @token = params[:token]
-    @nomination_email = NominationEmail.find_by(token: @token)
+  def start
+    token = params[:token]
+    nomination_email = NominationEmail.find_by(token: token)
 
-    if @nomination_email.nil?
+    if nomination_email.nil?
       redirect_to link_invalid_nominate_induction_coordinator_path
-    elsif @nomination_email.expired?
-      redirect_to link_expired_nominate_induction_coordinator_path(school_id: @nomination_email.school_id)
-    elsif @nomination_email.school.registered?
+    elsif nomination_email.expired?
+      redirect_to link_expired_nominate_induction_coordinator_path(school_id: nomination_email.school_id)
+    elsif nomination_email.school.registered?
       redirect_to already_nominated_request_nomination_invite_path
     else
       load_nominate_induction_tutor_form
-      @nominate_induction_tutor_form.token = @token
+      @nominate_induction_tutor_form.token = token
+      session[:nominate_induction_tutor_form] = @nominate_induction_tutor_form
```

Comment:
If you're saving the form in the session, I think we should delete it on success, just for completeness. Also, we wouldn't need to pass parameters to `#email_used`, we could just load the form and use the info from there

Reply:
Good point! Will do.

Reply:
@ethanmills I believe this should now be done :)